config/src/main/java/tech/pegasys/pantheon/config/IbftConfigOptions.java

```diff
@@ -36,7 +36,7 @@ public int getBlockPeriodSeconds() {
     return ibftConfigRoot.getInteger("blockperiodseconds", DEFAULT_BLOCK_PERIOD_SECONDS);
   }
 
-  public int getRequestTimeoutMillis() {
-    return ibftConfigRoot.getInteger("requesttimeout", DEFAULT_ROUND_EXPIRY_MILLISECONDS);
+  public int getRequestTimeoutSeconds() {
+    return ibftConfigRoot.getInteger("requesttimeoutseconds", DEFAULT_ROUND_EXPIRY_MILLISECONDS);
```

Comment:
still got DEFAULT_ROUND_EXPIRY_MILLISECONDS while we're now talking about seconds

Reply:
oops. Done.